src/css/forms.scss

```diff
@@ -45,7 +45,7 @@
 
 .form__row {
     display: block;
-    margin-bottom: 20px;
+    margin-bottom: 16px;
```

Comment:
This will be changing all the form__row used in dashboard. Is this change required everywhere?